src/container/internal.tsx

```diff
@@ -97,7 +99,7 @@ export default function InternalContainer({
         styles.root,
         styles[`variant-${variant}`],
         fitHeight && styles['fit-height'],
-        isSticky && [styles['sticky-enabled']]
+        isSticky && !isMobile && [styles['sticky-enabled']]
```

Comment:
I think it is not going to be obvious for future readers why we make this distinction. At minimum I would suggest to add a comment. Perhaps ideally there is room for some refactoring, by replacing the term "sticky" with "sticky header" or "sticky table header", depending on the case, for more clarity.

Reply:
Good point, I added a comment + a variable to hopefully make it more clear.